test/support/seededRuns.ts

```diff
@@ -295,6 +295,8 @@ class TestGenerator<
       .map(([key]) => key)
       .sort();
     vi_it('should test all methods', () => {
+      // @ts-expect-error: Argument of type 'string' is not assignable to parameter of type 'MethodOf<TModule>'.
```

Comment:
IMO this shouldn't tell what it hides but why.

```suggestion
      // @ts-expect-error: The method is protected and thus invisible in this scope
```

Reply:
Can you move this code to the actual spec file please?